src/App.js

```diff
@@ -1,23 +1,49 @@
-import logo from './logo.svg';
-import './App.css';
+import "./global.css";
+import Navbar from "./components/Navbar";
+import FolderInfoView from "./components/FolderInfoView";
+import FolderView from "./components/FolderView";
+import Footer from "./components/Footer";
+import { fetchSampleProfile, fetchSampleFolder } from "./api";
+import { useEffect, useState } from "react";
 
 function App() {
+  const [profile, setProfile] = useState(null);
+  const [folder, setFolder] = useState(null);
+
+  async function getProfile() {
+    let newProfile = null;
+    try {
+      newProfile = await fetchSampleProfile();
+    } catch (e) {
+      alert(e);
+      return;
+    }
+    setProfile(newProfile);
```

Comment:
굳이 let 변수로 선언해놓지 않아도 될 것 같아요.
아래와 같이 해도 동일하지 않을까요? 
try-catch 바깥에서 반드시 setState를 호출하지 않아도 괜찮을 것 같아요 
```suggestion
    try {
      const newProfile = await fetchSampleProfile();
      setProfile(newProfile);
    } catch (e) {
      alert(e);
      return;
    }
```